Dockerfile.openshift.rhel7

```diff
@@ -1,11 +1,10 @@
-FROM openshift/origin-release:golang-1.10 as build
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 WORKDIR /go/src/github.com/coredns/coredns
-ENV GOPATH=/go
 COPY . .
 RUN go build -o coredns .
 
-FROM centos:7
-COPY --from=build /go/src/github.com/coredns/coredns/coredns /usr/bin/coredns
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
+COPY --from=builder /go/src/github.com/coredns/coredns/coredns /usr/bin/coredns
```

Comment:
You can omit the trailing `/coredns` and just have `/usr/bin/` as the destination.

Reply:
(do that in both docker files

Reply:
Fixed